<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="Main_Layout">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QPushButton" name="Light_Import_BTN">
      <property name="text">
       <string>导入灯光Json文件</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QCheckBox" name="aiSkyDomeLight_CB">
        <property name="text">
         <string>aiSkyDomeLight</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="DirectionalLight_CB">
        <property name="text">
         <string>DirectionalLight</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="AreaLight_CB">
        <property name="text">
         <string>AreaLight</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="PointLight_CB">
        <property name="text">
         <string>PointLight</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="SpotLight_CB">
        <property name="text">
         <string>Spotlight</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="aiAreaLight_CB">
        <property name="text">
         <string>aiAreaLight</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Light_Export_BTN">
      <property name="text">
       <string>导出灯光Json文件</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;灯光类型过滤器 - 勾选说明允许导入导出&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="Main_Menu_Bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>313</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="Win_Menu">
    <property name="title">
     <string>窗口设置</string>
    </property>
    <addaction name="Maya_Dock"/>
    <addaction name="Maya_Undock"/>
    <addaction name="Maya_Workspace"/>
   </widget>
   <widget class="QMenu" name="Setting_Menu">
    <property name="title">
     <string>编辑设置</string>
    </property>
    <addaction name="Save_Setting"/>
    <addaction name="Load_Setting"/>
    <addaction name="Default_Setting"/>
   </widget>
   <widget class="QMenu" name="Help_Menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="Help_Instruction"/>
   </widget>
   <addaction name="Setting_Menu"/>
   <addaction name="Win_Menu"/>
   <addaction name="Help_Menu"/>
  </widget>
  <action name="Maya_Dock">
   <property name="text">
    <string>停靠</string>
   </property>
  </action>
  <action name="Maya_Undock">
   <property name="text">
    <string>不停靠</string>
   </property>
  </action>
  <action name="Maya_Workspace">
   <property name="text">
    <string>工作区</string>
   </property>
  </action>
  <action name="Save_Setting">
   <property name="text">
    <string>保存设置</string>
   </property>
  </action>
  <action name="Load_Setting">
   <property name="text">
    <string>读取设置</string>
   </property>
  </action>
  <action name="Default_Setting">
   <property name="text">
    <string>重置设置</string>
   </property>
  </action>
  <action name="More_Setting">
   <property name="text">
    <string>开启更多设置</string>
   </property>
  </action>
  <action name="Close_BTN">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="Help_Instruction">
   <property name="text">
    <string>使用说明</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
